scripts/generateRedirects.js

```diff
@@ -0,0 +1,34 @@
+const { resolve } = require('path');
+const { readFileSync, writeFile } = require('fs');
+const redirects = require('../util-node/redirects');
+
+const firebaseConfig = resolve(__dirname, '../firebase.json');
+
+const firebaseJSON = JSON.parse(readFileSync(firebaseConfig));
+
+let firebaseRedirects = [];
+
+firebaseRedirects = Object.entries(redirects).map(([key, value]) => ({
+  ...firebaseRedirects,
+  source: key,
+  destination: value,
+  type: '301',
+}));
+
+writeFile(
+  firebaseConfig,
+  JSON.stringify(
+    {
+      ...firebaseJSON,
+      redirects: firebaseRedirects,
+    },
+    null,
+    2
+  ),
+  err => {
+    if (err) console.error('error writing redirects', err);
+    else {
+      console.log('redirects written successfully\n');
```

Comment:
I don't think we need this

Reply:
Need consoles?

Reply:
It'd be better to catch these errors and show during build. Otherwise we'd never know?

Reply:
Adding comments to suppress eslint also seems ok in this case. However I don't see any reason for this to fail. We could still remove the `console.log` for success though. @harish-sethuraman

Reply:
Ive removed the success log